airbyte-webapp/src/locales/en.json

```diff
@@ -112,7 +112,7 @@
   "sources.schemaSelectAll": "select all",
   "sources.schemaUnselectAll": "unselect all",
   "sources.settings": "Settings",
-  "sources.syncHistory": "Sync history",
+  "sources.syncHistory": "Sync History",
```

Comment:
the lower-case h was really bothering me:

<img width="838" alt="Screen Shot 2021-03-11 at 7 39 36 PM" src="https://user-images.githubusercontent.com/9772859/110891186-720a2800-832c-11eb-8e45-8f01c1e21c7c.png">